autogpt_platform/backend/backend/executor/scheduler.py

```diff
@@ -418,12 +452,12 @@ def add_graph_execution_schedule(
             execute_graph,
             kwargs=job_args.model_dump(),
             name=name,
-            trigger=CronTrigger.from_crontab(cron),
+            trigger=CronTrigger.from_crontab(cron, timezone=user_timezone),
```

Comment:
I can pull the cron trigger conversion from non utc to utc out of here but it seems like as good a place as any for it